src/transformers/configuration_utils.py

```diff
@@ -436,6 +452,7 @@ def from_dict(cls, config_dict: Dict[str, Any], **kwargs) -> "PretrainedConfig":
             kwargs (:obj:`Dict[str, Any]`):
                 Additional parameters from which to initialize the configuration object.
 
+
```

Comment:
was this intentional to add all those new lines?

Reply:
They were added by `make style` !

Reply:
that's weird...what version do you have installed for black?

Reply:
19.10b0

Reply:
Can you update to `20.8b1`? Think that's the version we currently use. And isort should be 5.6.4 if I remember correctly